test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpath-plugin.yaml

```diff
@@ -354,7 +354,7 @@ spec:
               name: socket-dir
 
         - name: csi-snapshotter
-          image: registry.k8s.io/sig-storage/csi-snapshotter:v8.1.0
+          image: registry.k8s.io/sig-storage/csi-snapshotter:v8.2.0
           args:
             - -v=5
             - --csi-address=/csi/csi.sock
```

Comment:
Now that snapshots are v1beta1, I think it's time to enable the feature gate here in the external-snapshotter by default [here](https://github.com/kubernetes/kubernetes/blob/9709d36dda75ff82d1a8121edb23568a1d65ab82/test/e2e/storage/drivers/csi.go#L284-L291).

We could remove run_group_snapshot_e2e.sh and related manifests. A CI job would only need to ensure that the snapshot controller is started with the group snapshot feature gate.

Reply:
@xing-yang WDYT?

Reply:
plus group snapshot CRDs need to moved to cluster addons, as of now there are in external snapshotter

Reply:
ok, we should not add beta CRDs to the default addons. Let's use the script as it is until we find a better way how to run the tests.